blog/templates/blog/blog_posts_list.haml

```diff
@@ -24,7 +24,21 @@
                         .popular-article
                             %a.title{:href => "{% url 'wagtail_serve' article.specific.slug %}"} {{ article.specific.title }}
                             .author by
-                                %strong {{ article.specific.author.full_name }}
+                                %span.author-name{:onClick => 'showBio(this, "{{ article.specific.author.bio }}")'}
+                                    {{ article.specific.author.full_name }}
+                                    .bio-modal
+                                        .bio-content
+                                            %span.close &times
+                                            .bio-author
+                                                .bio-img
+                                                    {% thumbnail article.specific.author.front_image "400x400" as image %}
+                                                        %img.bio-img.lazyload{:data-src => "{{ image.url }}", :alt => "{{ article.specific.author.full_name }}"}
+                                                    {% endthumbnail %}
+                                                .bio-name
+                                                    {{ article.specific.author.full_name }}
+                                            .bio-text
+                                                .text
+                                                    {{ article.specific.author.bio|urlize }}
```

Comment:
I see that this is duplicated code with same thing in `blog/templates/blog/blog_posts_list_partials/main_article.haml`. This is perfect example to extract it to separate template and inject this template in this two places. Please change it

Reply:
I see that it is even in more than 2 places :) This is more than perfect example to extract it to separate template

Reply:
Done